tests/regression/common.sh

```diff
@@ -18,6 +18,17 @@ setup_build_opts() {
     fail=()
 }
 
+# Set sanitizer options
+setup_sanitizer_opts() {
+    SAN_OPTIONS=(
+        "--config=asan"
+        #"--config=tsan"
```

Comment:
Why comment out this config?

Reply:
Let me uncomment it, but fixing the corresponding will be deferred